atomic_reactor/util.py

```diff
@@ -852,4 +852,145 @@ def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, par
     return dfparser
 
 
+class Labels(object):
```

Comment:
How about making some class variables such as `LABEL_NAME=1`, `LABEL_COMPONENT=2`, etc, and having a get_preferred_label_value() method which knows about iterating through the preferred list of label names and checking for each one in turn.

Then get_name() etc could use it. In other words make it DRY - don't repeat yourself.

The common case is that we just want to know the value for a particular field and don't care which label provides it.

However we also need a method which provides access to the full list of labels relevant to a particular field. (I don't remember where that is required.) So that could be get_preferred_label_names().